convert2rhel/actions/conversion/list_non_red_hat_pkgs_left.py

```diff
@@ -27,6 +27,7 @@
 
 class ListNonRedHatPkgsLeft(actions.Action):
     id = "LIST_NON_RED_HAT_PKGS_LEFT"
+    dependencies = ("CONVERT_SYSTEM_PACKAGES",)
```

Comment:
Shouldn't it be the `FIX_DEFAULT_KERNEL`? (which I believe is the last from `preserve_only_rhel_kernel.py`)

The original order I use as source: https://github.com/oamg/convert2rhel/blob/v1.7.1/convert2rhel/main.py#L299

Maybe in the nonredhatpkgs could be listed some non-RHEL kernel which would be removed during `KernelPkgsInstall`

Reply:
That's true. Good point. 

In that case, I believe it would be `KERNEL_PACKAGES_INSTALLATION`, as we would have the installed kernel ready and all the dependencies from it as well. Will update the code

Reply:
```suggestion
    dependencies = ("KERNEL_PACKAGES_INSTALLATION",)
```